src/components/engagement/engagement.service.ts

```diff
@@ -982,15 +945,31 @@ export class EngagementService {
         canRead: canReadEndDate,
         canEdit: false,
       },
+      methodologies: {
+        ...securedProperties.methodologies,
+        value: securedProperties.methodologies.value ?? [],
+      },
+      ceremony: {
+        ...securedProperties.ceremony,
+        value: result.ceremonyId,
+      },
```

Comment:
Would be nice if there was a way to handle these "relationship" reference more easily. 🤔

Reply:
🤔🤔 seems simple enough already, what did you have in mind?

Reply:
Yeah idk It's good enough for now 👍